src/main/resources/org/broadinstitute/dsde/sam/liquibase/changesets/20220511_tos_version.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="dvoet" id="add_tos_version">
+        <addColumn tableName="sam_user">
+            <column name="accepted_tos_version" type="VARCHAR(40)">
+            </column>
+        </addColumn>
+    </changeSet>
+    <changeSet logicalFilePath="dummy" author="dvoet" id="migrate_tos">
+        <sql>
+            update sam_user set accepted_tos_version = '0' where id in
```

Comment:
It looks like when we create a user, accepted_tos_version is is set to `None`. Any reason why we initialize to 0 here?

it's small potatoes but for consistency, should we set it to NULL here?
```suggestion
            update sam_user set accepted_tos_version = NULL where id in
```

Reply:
no, this is a migration to set the version to '0' for anyone who has already accepted version 0.

Reply:
OH I see that now. Thanks!